2013-09-07 19:05:15 Error: Testing LogError() basic function
2013-09-07 19:05:15 Error: Testing with multi-line string
2013-09-07 19:18:02 Error: [UnexpectedValueException] Referenced fixture class Note (Test\Fixture\NoteFixture.php) not found
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\Fixture\CakeFixtureManager.php(79): CakeFixtureManager->_loadFixtures(Array)
#1 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestRunner.php(58): CakeFixtureManager->fixturize(Object(TaskTest))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteCommand.php(115): CakeTestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(247): CakeTestSuiteCommand->run(Array)
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(100): CakeTestSuiteDispatcher->_runTestCase()
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(117): CakeTestSuiteDispatcher->dispatch()
#6 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\test.php(100): CakeTestSuiteDispatcher::run()
#7 {main}
2013-09-07 20:12:33 Error: Fatal Error (1): Call to undefined function getLatest() in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Case\Model\TaskTest.php, line 28]
2013-09-07 20:12:33 Error: [FatalErrorException] Call to undefined function getLatest()
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'E:\ProgramFiles...', 28)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'E:\ProgramFiles...', 28, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'E:\ProgramFiles...', 28, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-07 20:14:38 Error: Fatal Error (1): Call to undefined method TaskTest::find() in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Case\Model\TaskTest.php, line 41]
2013-09-07 20:14:38 Error: [FatalErrorException] Call to undefined method TaskTest::find()
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'E:\ProgramFiles...', 41)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'E:\ProgramFiles...', 41, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'E:\ProgramFiles...', 41, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-07 20:16:52 Error: Fatal Error (1): Call to undefined method TaskTest::getLatest() in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Case\Model\TaskTest.php, line 28]
2013-09-07 20:16:52 Error: [FatalErrorException] Call to undefined method TaskTest::getLatest()
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'E:\ProgramFiles...', 28)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'E:\ProgramFiles...', 28, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'E:\ProgramFiles...', 28, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-07 20:23:44 Error: [MissingActionException] Action Tasks::app() could not be found.
Exception Attributes: array (
  'controller' => 'Tasks',
  'action' => 'app',
)
Request URL: /cakephp/Tasks/app/Test/Case/Model/TaskTest.php?debug_host=192.168.1.16%2C127.0.0.1&start_debug=1&debug_port=10000&original_url=http%3A%2F%2Flocalhost%2FTasks%2Fapp%2FTest%2FCase%2FModel%2FTaskTest.php&send_sess_end=1&debug_stop=1&debug_start_session=1&debug_no_cache=1378552919578&debug_session_id=1000
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Scaffold.php(153): Scaffold->_scaffold(Object(CakeRequest))
#1 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(531): Scaffold->__construct(Object(TasksController), Object(CakeRequest))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(490): Controller->_getScaffold(Object(CakeRequest))
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(TasksController), Object(CakeRequest), Object(CakeResponse))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2013-09-14 17:36:45 Error: Fatal Error (64): Cannot redeclare TaskFixture::$import in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\TaskFixture.php, line 6]
2013-09-14 17:36:45 Error: [FatalErrorException] Cannot redeclare TaskFixture::$import
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 17:37:12 Error: Fatal Error (64): Cannot redeclare TaskFixture::$import in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\TaskFixture.php, line 6]
2013-09-14 17:37:12 Error: [FatalErrorException] Cannot redeclare TaskFixture::$import
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 17:39:56 Error: Fatal Error (64): Cannot redeclare TaskFixture::$import in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\TaskFixture.php, line 6]
2013-09-14 17:39:56 Error: [FatalErrorException] Cannot redeclare TaskFixture::$import
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 17:44:01 Error: Fatal Error (64): Cannot redeclare TaskFixture::$import in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\TaskFixture.php, line 6]
2013-09-14 17:44:01 Error: [FatalErrorException] Cannot redeclare TaskFixture::$import
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 17:49:44 Error: [UnexpectedValueException] Referenced fixture class Note (Test\Fixture\NoteFixture.php) not found
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\Fixture\CakeFixtureManager.php(79): CakeFixtureManager->_loadFixtures(Array)
#1 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestRunner.php(58): CakeFixtureManager->fixturize(Object(TaskTest))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteCommand.php(115): CakeTestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(247): CakeTestSuiteCommand->run(Array)
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(100): CakeTestSuiteDispatcher->_runTestCase()
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\TestSuite\CakeTestSuiteDispatcher.php(117): CakeTestSuiteDispatcher->dispatch()
#6 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\test.php(100): CakeTestSuiteDispatcher::run()
#7 {main}
2013-09-14 17:51:55 Error: Fatal Error (64): Cannot redeclare class TaskFixture in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\NoteFixture.php, line 6]
2013-09-14 17:51:55 Error: [FatalErrorException] Cannot redeclare class TaskFixture
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 17:52:26 Error: Fatal Error (64): Cannot redeclare class TaskFixture in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Test\Fixture\NoteFixture.php, line 6]
2013-09-14 17:52:26 Error: [FatalErrorException] Cannot redeclare class TaskFixture
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-14 18:22:49 Error: Fatal Error (1): Class 'AppController' not found in [E:\ProgramFiles\xampp\htdocs\cakephp\app\Controller\TasksController.php, line 6]
2013-09-14 18:22:49 Error: [FatalErrorException] Class 'AppController' not found
Request URL: /cakephp/tasks/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Class 'AppContr...', 'E:\ProgramFiles...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AppContr...', 'E:\ProgramFiles...', 6, Array)
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(927): call_user_func('ErrorHandler::h...', 1, 'Class 'AppContr...', 'E:\ProgramFiles...', 6, Array)
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Core\App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-21 18:07:18 Error: [MissingActionException] Action Tasks::sql.php() could not be found.
Exception Attributes: array (
  'controller' => 'Tasks',
  'action' => 'sql.php',
)
Request URL: /cakephp/tasks/sql.php
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Scaffold.php(153): Scaffold->_scaffold(Object(CakeRequest))
#1 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(531): Scaffold->__construct(Object(TasksController), Object(CakeRequest))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(490): Controller->_getScaffold(Object(CakeRequest))
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(TasksController), Object(CakeRequest), Object(CakeResponse))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2013-09-21 18:42:31 Error: [MissingActionException] Action Tasks::Sql() could not be found.
Exception Attributes: array (
  'controller' => 'Tasks',
  'action' => 'Sql',
)
Request URL: /cakephp/tasks/Sql/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Scaffold.php(153): Scaffold->_scaffold(Object(CakeRequest))
#1 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(531): Scaffold->__construct(Object(TasksController), Object(CakeRequest))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(490): Controller->_getScaffold(Object(CakeRequest))
#3 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(TasksController), Object(CakeRequest), Object(CakeResponse))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2013-09-21 20:31:17 Error: [MissingControllerException] Controller class CsrtController could not be found.
Exception Attributes: array (
  'class' => 'CsrtController',
  'plugin' => NULL,
)
Request URL: /cakephp/Csrt
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-21 20:31:36 Error: [MissingControllerException] Controller class Csrt.htmlController could not be found.
Exception Attributes: array (
  'class' => 'Csrt.htmlController',
  'plugin' => NULL,
)
Request URL: /cakephp/Csrt.html
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-21 20:40:03 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/csrf
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2013-09-21 20:46:53 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/csrf
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2013-09-21 20:47:30 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/Csrf/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2013-09-21 20:59:03 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/Csrf/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2013-09-21 21:00:40 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/Csrf/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2013-09-21 21:01:26 Error: [BadRequestException] The request has been black-holed
Request URL: /cakephp/Csrf/index
Stack Trace:
#0 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Component\SecurityComponent.php(244): SecurityComponent->blackHole(Object(CsrfController), 'csrf')
#1 [internal function]: SecurityComponent->startup(Object(CsrfController))
#2 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Utility\ObjectCollection.php(132): call_user_func_array(Array, Array)
#3 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#4 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Event\CakeEventManager.php(248): call_user_func(Array, Object(CakeEvent))
#5 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#6 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(184): Controller->startupProcess()
#7 E:\ProgramFiles\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(CsrfController), Object(CakeRequest), Object(CakeResponse))
#8 E:\ProgramFiles\xampp\htdocs\cakephp\app\webroot\index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
